### DISCUSSION TITLE
Should these two languages have the same color?

### DISCUSSION THREAD

**Discussion by DecimalTurn** [github-linguist/linguist#6189] [General]
I noticed that `VBA` an `FreeBasic` have the same color
https://github.com/github/linguist/blob/59b2d88b2242e6062384e5fb876668cc30ead951/lib/linguist/languages.yml#L1996-L1998

https://github.com/github/linguist/blob/59b2d88b2242e6062384e5fb876668cc30ead951/lib/linguist/languages.yml#L6976-L6978

@XusinboyBekchanov, as the author of https://github.com/github/linguist/pull/4998, can you clarify if it was intentional to use the same color as VBA?
Those languages have similarities, but I don't think it's enough to justify using the same color.

**Comment by XusinboyBekchanov** [comment]
I just copied the right settings, the color for me was not interesting at that time. FreeBasic used to be defined as VBA with this color.

**Comment by DecimalTurn** [comment]
OK, thank you for your answer. That's what I suspected. 

@lildude, would you say that, in this case, the requirements for changing a language color expressed in the [Contributing guidelines](https://github.com/github/linguist/blob/master/CONTRIBUTING.md#changing-the-color-associated-with-a-language) can be waived?

If yes, I would go ahead and propose to change FreeBasic's color for a blue inspired by the logo at the top of this [FreeBasic and QBasic forum](https://forum.qbasic.at/) : `#141AC9`

![image](https://user-images.githubusercontent.com/31558169/203892894-15374038-0e2b-4621-8596-58c0ba0d12e8.png)

There is also some other logos discussed in [this thread](https://www.freebasic.net/forum/viewtopic.php?t=12050).
This one is a lighter blue which could also work.

![image](https://user-images.githubusercontent.com/31558169/203894181-0f4156b5-2562-4313-80e3-41fa596466e2.png)

**Reply by XusinboyBekchanov** [reply]
Started a discussion here too:
https://freebasic.net/forum/viewtopic.php?p=295775#p295775

**Reply by lildude** [reply]
> @lildude, would you say that, in this case, the requirements for changing a language color expressed in the [Contributing guidelines](https://github.com/github/linguist/blob/master/CONTRIBUTING.md#changing-the-color-associated-with-a-language) can be waived?

Yes, I think we can make an exception here, especially as the new colour isn't vastly different so isn't likely to raise objections.

**Reply by XusinboyBekchanov** [reply]
The suggested color is perfect for me too.

**Comment by jayrm** [comment]
@XusinboyBekchanov, thank-you for posting on our forum.  To be honest, I'm not sure we will get much response.  My feeling is that only a handful of developers actually use github, and they probably never even thought about colour display on github, including myself.

@DecimalTurn, your proposal for colour is acceptable to me.  And it's very likely that I may be the only developer to respond.
